test/OpenIddict.Server.Tests/OpenIddict.Server.Tests.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;net472;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\OpenIddict.Abstractions\OpenIddict.Abstractions.csproj" />
```

Comment:
This package is reference by `OpenIddict.Server` so you should be able to remove it.

Reply:
Yeah, good call. I added the Abstractions reference first while going through the 'fix all the red squigglies from namespaces that could not be found'. Didn't think about it being a transitive dependency.

Updated.